### ISSUE TITLE
Install fails because version.get_version fails

### ISSUE THREAD

**Issue by john-g-g** [google/pygtrie#13]
### pip install from pypi fails
```
$ mktmpenv
Using base prefix '/usr/local/Cellar/python3/3.5.2_3/Frameworks/Python.framework/Versions/3.5'
New python executable in /Users/muyloco/Dev/Envs/tmp-5c77e50baa5495b6/bin/python3.5
Also creating executable in /Users/muyloco/Dev/Envs/tmp-5c77e50baa5495b6/bin/python
Installing setuptools, pip, wheel...done.
This is a temporary environment. It will be deleted when you run 'deactivate'.
(tmp-5c77e50baa5495b6) muyloco at cambria in ~/Dev/Envs/tmp-5c77e50baa5495b6
$ pip install pygtrie
Collecting pygtrie
  Using cached pygtrie-2.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-build-ap34m_ri/pygtrie/setup.py", line 6, in <module>
        import version  # pylint: disable=relative-import
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-build-ap34m_ri/pygtrie/version.py", line 128
        print get_version()
                        ^
    SyntaxError: invalid syntax
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-build-ap34m_ri/pygtrie/
(tmp-5c77e50baa5495b6) muyloco at cambria in ~/Dev/Envs/tmp-5c77e50baa5495b6
```
### pip install from master branch fails 
```
$ pip install git+https://github.com/google/pygtrie.git@master
Collecting git+https://github.com/google/pygtrie.git@master
  Cloning https://github.com/google/pygtrie.git (to master) to /private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-d79a4kzy-build
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-d79a4kzy-build/setup.py", line 8, in <module>
        release = version.get_version()
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-d79a4kzy-build/version.py", line 122, in get_version
        raise ValueError('Cannot find the version number')
    ValueError: Cannot find the version number
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-d79a4kzy-build/
```

### pip install from local repo fails
```
$ git clone https://github.com/google/pygtrie.git
Cloning into 'pygtrie'...
remote: Counting objects: 247, done.
remote: Compressing objects: 100% (27/27), done.
remote: Total 247 (delta 11), reused 0 (delta 0), pack-reused 220
Receiving objects: 100% (247/247), 150.25 KiB | 0 bytes/s, done.
Resolving deltas: 100% (135/135), done.
(tmp-5c77e50baa5495b6) muyloco at cambria in ~/Dev/Envs/tmp-5c77e50baa5495b6
$ cd pygtrie
$ git describe
v2.0-14-g2785d76
(tmp-5c77e50baa5495b6) muyloco at cambria in ~/Dev/Envs/tmp-5c77e50baa5495b6/pygtrie on master*
$ git describe --long --match 'v[0-9]*.*'                                                                                                                                                                                                   
v2.0-14-g2785d76
(tmp-5c77e50baa5495b6) muyloco at cambria in ~/Dev/Envs/tmp-5c77e50baa5495b6/pygtrie on master*
$ pip install .
Processing /Users/muyloco/Dev/Envs/tmp-5c77e50baa5495b6/pygtrie
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-573kfusl-build/setup.py", line 8, in <module>
        release = version.get_version()
      File "/private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-573kfusl-build/version.py", line 122, in get_version
        raise ValueError('Cannot find the version number')
    ValueError: Cannot find the version number
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/dm/nyn5y95d4mqbwvcz1jsgpg8w0000gn/T/pip-573kfusl-build/
```

**Comment by mina86** [comment]
I haven’t yet pushed the new version with Python 3.0 compatibility to PyPi so that’s why the first one fails. The latter two are due to my stupidity; thanks for the fix in #14.

**Comment by john-g-g** [comment]
Happy to help, #14 was much easier to write than the package :smile:
